commands/create-project-api.js

```diff
@@ -70,7 +73,7 @@ async function updatePackageJson(packageJson, projectName) {
  */
 function updateEnv(envData) {
   const env = parse(envData);
-  env.MONGO_URL = "mongodb://localhost:27017/reaction";
+  env.MONGO_URL = "mongodb://localhost:27017/test";
```

Comment:
This change the permanent link and cause production data and test data to comingle. This change needs to just change the the MONGO for the test runner only

Reply:
In the jest config, there is no reference for MONGO . Should I refer to some other location ?

Reply:
Hi @zenweasel I need some help regarding setting up of MONGO URL for test runner.  Can you please provide some direction

Reply:
Jest has a way to set ENV VARS just for the duration of the test run that is not specific to this one. You just need to apply that method, I don't know it off the top of my head.

Reply:
You need to update after you copy it into the project, not touch the original project files

Reply:
Is it fine if I completely update the file with :-

`
process.env = Object.assign(process.env, {
  MAIL_URL: "smtp://user:pass@email-smtp.us-west-2.amazonaws.com:465",
  REACTION_LOG_LEVEL: "ERROR",
  REACTION_WORKERS_ENABLED: false,
MONGO_URL: "mongodb://localhost:27017/test"
});

process.on("unhandledRejection", (err) => {
  throw err;
});
`

The only downside of it is that in the source file content changes, we may have compatibility issues.
Other solution is in the .env file we can use a new property like:-
`
  MONGO_TEST_URL: "mongodb://localhost:27017/test"
`

Reply:
I would suggest just adding the MONGO_URL entry after copying the file over

Reply:
The challenge is around adding it to a javascript file.  If we would have imported a json file & used it internally, it would have been very easy.

Reply:
Yeah, probably the least worst solution is to add the file with the change to the project template and just use that.

Reply:
instead should I create a pull request in **reaction** to import a json [here](https://github.com/reactioncommerce/reaction/blob/trunk/apps/reaction/tests/util/setupJestTests.js#L5) & use it like:-

`
import { createRequire } from "module";
const require = createRequire(import.meta.url);
const jestConfigs = require("./jestConfigs.json");
process.env = Object.assign(process.env, {
  ...jestConfigs
});

process.on("unhandledRejection", (err) => {
  throw err;
});
`
We can use it later in our repository & simply update the `jestConfigs.json`
It will help to reduce the maintenance overhead in cli

Reply:
I updated the template here:-
https://github.com/reactioncommerce/cli/compare/trunk...aditya-padhi-kbl:cli:trunk?expand=1#diff-68b75959b4ea738c8e466b862b12daea4e18d213e2c1f15884f68977c6b4fc7e

I feel if we merge https://github.com/reactioncommerce/reaction/pull/6568 then we don't have to maintain test template on our side

Reply:
@zenweasel 
Can you please review this [PR](https://github.com/reactioncommerce/reaction/pull/6568) & merge if the changes look good.  Once done I will go ahead & make changes to MONGO_URL integration

Reply:
Hi @zenweasel ,

Please review the changes. Mongo_url integration & other changes are done